Sources_v3/StreamChatUI/ChatChannel/Attachments/ChatMessageGiphyView.swift

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Nuke
+import StreamChat
+import SwiftyGif
+import UIKit
+
+open class ChatMessageGiphyView<ExtraData: ExtraDataTypes>: View, UIConfigProvider {
```

Comment:
nit: in the design the gif displaying component has `GIPHY` label. Would be nice to have it, but that's not super important

Reply:
My bad, totally missed it.

Reply:
c230616